File: 1/cf56093/task06/mem_usage_fluxline.js
var os = require('os');
var totalmem=os.totalmem();
var freemem=os.freemem();

var hrTime = process.hrtime.bigint()


console.log('mem_usage,totalmem='+totalmem+',freemem='+freemem+',mem_usage_percent='+Math.round((totalmem-freemem)/totalmem*100)+' '+Date.now()+'000000') 

File: 1/cf56093/task06/mem_usage_fluxline.py
import time
from psutil import virtual_memory

memory_info=virtual_memory();


print('mem_usage,totalmem='+str(memory_info[0])+',freemem='+str(memory_info[4])+',mem_usage_percent='+str(round((memory_info[0]-memory_info[4])/memory_info[0]*100))+' '+str(time.time_ns()));
